msteams-platform/samples/integrating-web-apps.md

```diff
@@ -36,6 +36,7 @@ Your Teams app must include required and expected collaborative features. To wor
 |Message external services  |[Bots](../bots/what-are-bots.md)<br/>[Outgoing Webhooks](../webhooks-and-connectors/what-are-webhooks-and-connectors.md)  |
 |Modals  |[Task modules](../task-modules-and-cards/what-are-task-modules.md)  |
 |Content-rich cards  |[Adaptive Cards](../task-modules-and-cards/what-are-cards.md)  |
+|Multi-app support  |[Extend across Microsoft 365](../m365-apps/overview.md)  |
```

Comment:
I like the idea of adding to the table, but I'd argue that multi-hub app support isn't really an "app feature" so much as a capability (other column). Let's remove this row, and edit these rows like this:

|Common app features   |Teams platform capabilities   |
|----------|-----------|
|Embedded webpage, homepage, or webview  |[Tabs](../tabs/what-are-tabs.md)<br/> [Tabs extended across Microsoft 365](../m365-apps/overview.md)|
|Share shortcuts and extensions  |[Message extensions](../messaging-extensions/what-are-messaging-extensions.md)<br/> [Message extensions for Outlook](../m365-apps/overview.md)  |